com.unity.netcode.gameobjects/Runtime/Transports/UTP/UnityTransport.cs

```diff
@@ -421,6 +419,17 @@ private void DisposeInternals()
             m_SendQueue.Clear();
         }
 
+        ~UnityTransport()
```

Comment:
I'm a bit scared to see relying on GC disposal being a fix rather than explicit .Dispose()

Do you think we don't call .Dispose() where we should? If so, I'm even more scared :(

Reply:
I'm sad but I understand :P